libportal/portal-qt5.h

```diff
@@ -32,6 +32,11 @@ XdpParent *xdp_parent_new_qt (QWindow *window);
 
 namespace XdpQt {
 
+// Returns a global instance of XdpPortal object and takes care
+// of its deletion
+XDP_PUBLIC
+XdpPortal *globalPortalObject();
```

Comment:
I think the XdpPortal object is meant to stay alive until the app is closed, not only for one-time use.

This way I can use it like:
`xdg_portal_call(XdpQt::globalPortalObject(), xdpParent, param1, param2...);`

With your suggestion (an example returning XdgPortal class wrapping a pointer):
`xdg_portal_call(XdpQt::createPortalObject().get(), xdpParent, param1, param2...);`
This will call `xdp_portal_create()` for every single call and delete it everytime.

Reply:
Or did you mean just keeping around XdgPortal class wrapper doing things automatically instead of a pointer to XdgPortal which we need to delete manually later? Isn't it better to just keep passing a pointer to global instance and not worry about keeping it around?

Reply:
@aleixpol what do you think will be the best solution then?

Reply:
just `XdpPortal globalPortalObject();`

If you want a suggestion of what XdpPortal would look like, I can provide it.

Reply:
> just `XdpPortal globalPortalObject();`
> 
> If you want a suggestion of what XdpPortal would look like, I can provide it.

Yes, please. The name `XdgPortal` is already taken btw.

Reply:
As I was looking into this, I realised I don't actually know what `xdp_parent_new_qt` is supposed to be for. Is it so we don't have wrappers for every single libportal API and just can use libportal with QWindows?

Reply:
As you know, each portal call takes `parent_window` option as a string and `xdp_parent_new_qt()` is meant to give you exactly that from QWindow, just wrapped into `XdpParent` object.

Reply:
I guess I misread the situation then. Sorry for the delay on the Pull Request.